test-framework/core/src/main/java/org/keycloak/testframework/server/KeycloakServerConfigBuilder.java

```diff
@@ -183,6 +200,10 @@ public Set<Dependency> toDependencies() {
         return dependencies;
     }
 
+    public Set<Path> toConfigFiles() {
```

Comment:
Shouldn't be available publicly:

```suggestion
    Set<Path> toConfigFiles() {
```

Reply:
Should I also remove the `public` keyword from the `toDependencies()` and `toArgs()` method?

Reply:
Good point, if you could remove those that'd be great

Reply:
Done!